scripts/superdesk/keyboard/keyboard.js

```diff
@@ -330,6 +330,37 @@
                 elt['on' + type] = false;
             }
         };
+
+        this.registerShortcut = function (hotkey, callback, options, elem) {
+            this.register(hotkey[0], gettext(hotkey[2]), hotkey[1]);
```

Comment:
ok so, do you have some idea how to initialize all shortcuts on app load? Or we need to define them separately? Is it better to define all of them in one place, like in one service, or as it was now, on every page were you call shortcut you also call a registration of that shortcut with `.run`?